dawn/src/dawn/Optimizer/PassManager.cpp

```diff
@@ -13,6 +13,7 @@
 //===------------------------------------------------------------------------------------------===//
 
 #include "dawn/Optimizer/PassManager.h"
+#include "dawn/IIR/ASTVisitor.h"
```

Comment:
weird that you need to add an include in a file that you did not touch

Reply:
That's really weird ... I don't remember adding this, maybe it's a side effect of merging?